torch/csrc/jit/codegen/cuda/scheduler/normalization.cpp

```diff
@@ -398,19 +411,63 @@ std::shared_ptr<ReductionParams> innerPersistentHeuristic(
       : bdimx + (device_warp_size - bdimx % device_warp_size);
 
   bool pad_bdimx = bdimx > 16 &&
-      padded_bdimx * bdimy * bdimz <
-          (int64_t)at::cuda::getCurrentDeviceProperties()->maxThreadsPerBlock;
+      padded_bdimx * bdimy * bdimz < (int64_t)dev_prop->maxThreadsPerBlock;
 
   pad_bdimx = pad_bdimx &&
       bdimx * inner_reduction_unroll_factor != inner_most_dimension_numel;
 
+  int64_t nvrtc_register_per_thread = LaunchParams::UNINITIALIZED_VAL;
```

Comment:
Please add a comment on how and why you're doing the below.

Reply:
added.